nodebuilder/node/admin.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/celestiaorg/celestia-node/libs/authtoken"
 )
 
-const APIVersion = "v0.2.2"
+const APIVersion = "v0.11.0"
```

Comment:
So now we have to PR a string change every time we update the version. I wonder if there is a way to automate that.

Reply:
Dont think so unfortunately :/ This is one unforeseen issue with #2768